scripts/install_poetry.py

```diff
@@ -938,6 +938,12 @@ def main():
     if args.uninstall or string_to_bool(os.getenv("POETRY_UNINSTALL", "0")):
         return installer.uninstall()
 
+    preexisting_venv = os.environ.get("VIRTUAL_ENV")
+    if preexisting_venv:
+        # We need to deactivate the existing virtual environment, if it exists, in case the contributor
+        # switching from elsewhere in the Vellum ecosystem
+        os.system("deactivate")
```

Comment:
during onboarding, @peicodes had his venv activated from vellum, so this will forcibly deactivated before continuing with the rest of setup